spectractor/simulation/adr/adr_utils_Spectractor.py

```diff
@@ -0,0 +1,97 @@
+import matplotlib.pyplot as plt
+import astropy.coordinates as AC
+from astropy import units as u
+import spectractor.simulation.adr.adr as ADR

```

Comment:
no need to import spectractor functions.
rename file into adr_utils.py

Reply:
what functions ?

Reply:
rename file into adr_utils.py: done